packages/web-components/src/components/va-text-input/va-text-input.tsx

```diff
@@ -84,6 +84,12 @@ export class VaTextInput {
     | 'text'
     | 'url';
 
+  /**
+   * The step attribute is a number, or the string 'any', that specifies the granularity of the value. For example: `<va-text-input type="number" step=".1"/>` enables float/decimal values to be valid and increment by one-tenth. <br/>
+   * Defaults to 1 for every field type except for time and datetime-local which default to 60 (seconds)
+   */
+  @Prop() step?: string;
```

Comment:
If we're adding a new prop, is it possible to write a test for it?

Reply:
I added some tests, but also decided to write a method to default the value to .01 when type="number", inputmode="decimal" and step is undefined.